### DISCUSSION TITLE
Dipping in again

### DISCUSSION THREAD

**Discussion by Ulukai55** [redeclipse/base#1209] [General]
Hi all, Ulukai here!

You may or may not remember me from years ago, but I recently picked up RE2 again on Steam and wow, time didn't stand still while I was gone... :D You made the move to the tesseract engine, did a menu overhaul and added so much to the game. It felt great to play some matches (usually in the evening, GMT+1 time) and I was happy to see I didn't lose the feel of the game after all these years. 

I see on the tracker that you have some very nice new maps and remixes cooking up, so I'm already looking forward to the next release. 

On a personal note, I've been very busy at work and my kids are growing up, so priorities have shifted. We're planning to redecorating our first floor of the house and I've started building our house in RE2 engine to see what colors would look like on our walls and how we should organise things :D. Most of the mapping basics are still there, but stuff like shaders and texture manipulating definitely need a refresh. 

I need to create windows now, but don't want to fall back on the default Glass materials because they can't be shaped into forms, however I've failed miserably to just select some geometry with a plain generic texture and turn that transparant. I tried with valpha, but that didn't work out. Can someone refresh my memory?

Oh, and how can I move the cubes in my selection grid again? I can't figure it out for the hell of me and resort to copy/delete/paste instead which isn't very practical. 

Thanks and looking forward to seeing you sometime online!

**Comment by Jigoku** [comment]
Hey Ulukai, nice to see you :wave: 
> I've started building our house in RE2 engine to see what colors would look like on our walls and how we should organise things :D

That's a really cool idea lol.

For the glass windows, try using alpha material, along with `vrefract` and `valpha` which will give a glassy look. 
Also, you can hold U whilst using rightclick to drag the grid, and everything within should move when you let go of U.

**Comment by Ulukai55** [comment]
Hi there Jigoku. Brilliant! Got it working now, I totally overlooked the alpha material, it all makes sense now :-D. Moving with U works indeed, not even needed to drag with the right mouse button. Just select, then hold U and move the mouse, let go.

If ever I decide to create a playable map of it, I'll make sure to share it here, just for the fun of it to give you a virtual look into my home. :) I bet my kids would love to shoot some bots in our garden.

**Comment by SniperGoth** [comment]
Oh, look who is back! Welcome to the era of pretty lighting, and moving entities, hahah. :D

**Comment by IceflowRE** [comment]
Welcome back, you are one of the dinosaurs in the Red Eclipse history 🎉

**Comment by qreeves** [comment]
👋 We're on Discord now if you want to come hang out: https://www.redeclipse.net/discord

**Comment by Ulukai55** [comment]
Thanks all, glad to see some old names, you really keep this project going! I'll make sure to swing by on discord one of these days. 

> Oh, look who is back! Welcome to the era of pretty lighting, and moving entities, hahah. :D

You forgot killer rollers. I tried a match of onslaught and didn't stand a chance! :D I really like the improved sound effects for explosions and blasts too. The SMG recoil is really of this world though, being such a small machine gun lol. I was like WTF when I first used it.

**Comment by qreeves** [comment]
Yeah, that's fixed in the development version somewhat. On Steam you can go to the game properties and under "Betas" you can switch to the development branch easily.

**Comment by Ulukai55** [comment]
The house is coming along nicely. Some first images, all still very WIP. The bathroom is the only fully detailed room so far, I'm amazed how realistic I can recreate everything with the textures present in the game. 
Really enjoying the editing in the new engine. I actually really missed it, I'm burning through my free time so fast...

![Screenshot from 2020-09-19 23-54-52](https://user-images.githubusercontent.com/18656955/93690120-17a6b200-fad5-11ea-9a53-587820e3b67b.png)
![Screenshot from 2020-09-19 23-56-07](https://user-images.githubusercontent.com/18656955/93690121-18d7df00-fad5-11ea-9e71-871860bd8c67.png)
![Screenshot from 2020-09-19 23-56-50](https://user-images.githubusercontent.com/18656955/93690122-19707580-fad5-11ea-8f12-1939d5281c99.png)
![Screenshot from 2020-09-19 23-57-33](https://user-images.githubusercontent.com/18656955/93690123-1a090c00-fad5-11ea-945f-8d537c1f2a22.png)

**Comment by qreeves** [comment]
Haha, that's nice. Definitely shows we've collected a nice palette over the years for the artist to use 😄
